src/clj/web/lobby.clj

```diff
@@ -254,13 +254,14 @@
 (defn handle-lobby-create
   [{{{:keys [username] :as user} :user} :ring-req
     client-id                           :client-id
-    {:keys [title format allow-spectator save-replay spectatorhands password room side]} :?data :as event}]
+    {:keys [title format allow-spectator api-access save-replay spectatorhands password room side]} :?data :as event}]
```

Comment:
Maybe this happens elsewhere/as a matter of course, but should we check the that the user has-api-keys on the backend against the value passed from the frontend?

Reply:
I'm really only using the `has-keys` to make the UI clearer (so someone doesn't check the box and not know how to make it work). Any real authorization will happen when a request comes in on the (yet-to-be-written) API.

Reply:
Maybe another point that isn't clear - you need to have an API key and check the box in the Create Game screen to make the API work. So you can create games without allowing access with the API key.